frontend/src/views/Secrets.vue

```diff
@@ -6,127 +6,82 @@ SPDX-License-Identifier: Apache-2.0
-    @toogle-help="onToogleHelp"
-    @update="onUpdate"
-    @delete="onDelete"
-    ></secret>
-
-    <secret
-    v-if="hasCloudProfileForCloudProviderKind('openstack')"
-    class="mt-4"
-    infrastructure-key="openstack"
-    infrastructure-name="OpenStack"
-    icon="openstack"
-    description="Make sure that the new credentials have the correct OpenStack permissions"
-    @add="onAdd"
-    @toogle-help="onToogleHelp"
-    @update="onUpdate"
-    @delete="onDelete"
-    ></secret>
-
-    <secret
-    v-if="hasCloudProfileForCloudProviderKind('alicloud')"
-    class="mt-4"
-    infrastructure-key="alicloud"
-    infrastructure-name="Alibaba Cloud"
-    secret-descriptor-key="accessKeyID"
-    icon="alicloud"
-    description="Make sure that the new credentials have the correct Alibaba Cloud permissions"
-    @add="onAdd"
-    @toogle-help="onToogleHelp"
-    @update="onUpdate"
-    @delete="onDelete"
-    ></secret>
-
-    <secret
-    v-if="hasCloudProfileForCloudProviderKind('metal')"
-    class="mt-4"
-    infrastructure-key="metal"
-    infrastructure-name="Metal Cloud"
-    secret-descriptor-key="metalHMAC"
-    icon="metal"
-    description="Make sure that the new credentials have the correct Metal Cloud permissions"
-    @add="onAdd"
-    @toogle-help="onToogleHelp"
-    @update="onUpdate"
-    @delete="onDelete"
-    ></secret>
-
-    <secret
-    v-if="hasCloudProfileForCloudProviderKind('vsphere')"
-    class="mt-4"
-    infrastructure-key="vsphere"
-    infrastructure-name="VMware vSphere"
-    secret-descriptor-key="vsphereUsername"
-    icon="vsphere"
-    description="Make sure that the new credentials have the correct VMware vSphere permissions"
-    @add="onAdd"
-    @toogle-help="onToogleHelp"
-    @update="onUpdate"
-    @delete="onDelete"
-    ></secret>
-
-    <template v-if="showDisabledCloudProviders">
-
-      <disabled-secret
-      class="mt-4"
-      infrastructure-name="Digital Ocean"
-      icon="digital-ocean"
-      description="Before you can provision and access a Kubernetes cluster on Digital Ocean, you need to add account credentials."
-        ></disabled-secret>
-
-      <disabled-secret
-      class="mt-4"
-      infrastructure-name="China Telecom"
-      icon="china-telecom"
-      description="Before you can provision and access a Kubernetes cluster on China Telecom, you need to add account credentials."
-        ></disabled-secret>
-
-      <disabled-secret
-      class="mt-4"
-      infrastructure-name="Nutanix"
-      icon="mdi-xamarin"
-      description="Before you can provision and access a Kubernetes cluster on Nutanix, you need to add account credentials."
-        ></disabled-secret>
-
-    </template>
+      <v-card-text v-if="!secretList.length">
+        <div class="title grey--text text--darken-1 my-4">Add Secrets to your project.</div>
+        <p class="body-1">
+          Add Secrets to your project. You need to create at least one secret to be able to create a cluster.
```

Comment:
<img width="1081" alt="Screenshot 2021-01-29 at 16 11 09" src="https://user-images.githubusercontent.com/5526658/106292058-b0e89080-624c-11eb-8afe-72f3feabe5df.png">

> Before you can provision and access a Kubernetes cluster, you need to add account credentials.
\<CREATE\>

Maybe we should also hide the table when there are no secrets?

Reply:
Yes, good point. I will implement it in the same way as we have it for the member section.

Reply:
done